cypress/integration/homework/calculator.spec.js

```diff
@@ -0,0 +1,158 @@
+// <reference types="Cypress" />
+
+import HomePage from './classes/HomePage.js'
+
+var homePage = new HomePage
+
+var website = "https://testsheepnz.github.io/BasicCalculator";
+
+var numberField1 = "#number1Field";
+var numberField2 = "#number2Field";
+var clearButton = "#clearButton";
+var buildDropDown = "#selectBuild";
+var actionDropDown = "#selectOperationDropdown";
```

Comment:
As I see, you have all page objects in your _POM_ file (integration/homework/classes/HomePage.js). Think about removing them from here and using only in page object file ;)